sale_delivery_state/models/sale_order.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Pierrick BRUN <pierrick.brun@akretion.com>
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_compare, float_is_zero
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    delivery_state = fields.Selection(
```

Comment:
this module seems to clash or overlap w/ https://github.com/OCA/sale-workflow/blob/14.0/sale_automatic_workflow/models/sale_order.py#L18

Could you change that module to depends on this and refactor that field to be computed on `delivery_state == done`?

Computing the same info and storing it twice seems useless.